Map/MapManager.gd

```diff
@@ -2,27 +2,28 @@ extends Node2D
 ## Class to manage backend for rooms (generation and such)
  
 #layout changes the width of the map's floors
-static func create_map(_layout: Array[int]) -> MapBase: ## Generates and Populates a map with rooms that have random room types. More in depth algorithms will be added in the future
+static func create_map(floors_width: Array[int]) -> MapBase: ## Generates and Populates a map with rooms that have random room types. More in depth algorithms will be added in the future
```

Comment:
Turtyo has a point, we have floors_width already defined in MapBase, so it may be beneficial to update this name so we don't run into any complications once we add this to the scene.